templates/mainapp/layout.html

```diff
@@ -39,7 +39,7 @@
     <header>
         <div id="topheader">
             <nav class="navbar navbar-expand-md navbar-light">
-                <a class="navbar-brand head-text" href="/"><img width="100px" src="{% static 'mainapp/png/brand-img.png' %}"></a>
+                <a class="navbar-brand head-text" href="/"><img class="navbrandimg" src="{% static 'mainapp/png/brand-img.png' %}"></a>
```

Comment:
As the head-text class is not needed now remove it and also associated CSS with it.